java/vector/src/main/codegen/templates/FixedValueVectors.java

```diff
@@ -39,20 +39,26 @@
  * ${minor.class} implements a vector of fixed width values.  Elements in the vector are accessed
  * by position, starting from the logical start of the vector.  Values should be pushed onto the
  * vector sequentially, but may be randomly accessed.
+<#if (type.width > 0) >
```

Comment:
I think we should try to minimize the if else statement in the template to simply. I prefer we remove this if else here and state something like:

```
The width of each element is defined in getWidth()
```